### DISCUSSION TITLE
I'm not sure if it's a bug: xmake checks path `/usr/local/cuda/bin/usr/local/cuda/bin/nvcc`

### DISCUSSION THREAD

**Discussion by owlx56** [xmake-io/xmake#4765] [Q&A]
I'm using xmake in VSCode to write cuda code. Every time I change `xmake.lua` file and run the project, xmake outputs in terminal:

```
checking for /usr/local/cuda/bin/usr/local/cuda/bin/nvcc ... no 
```

![image](https://github.com/xmake-io/xmake/assets/105162283/f13d68a5-8722-45d1-a41d-9d8142f6adfa)

The project can be built normally, I am just confused about the output: Why does xmake check path `/usr/local/cuda/bin/usr/local/cuda/bin/nvcc`?

My `compile_commands.json`:

```
[
    {
        "directory": "/home/sirius/code/cpp/cuda-test",
        "arguments": [
            "/usr/local/cuda/bin/nvcc",
            "-c",
            "-Xcompiler",
            "-fPIE",
            "-O3",
            "-Isrc",
            "-I/usr/local/cuda/include",
            "-m64",
            "-rdc=true",
            "-gencode",
            "arch=compute_86,code=sm_86",
            "-o",
            "build/.objs/cuda-test/linux/x86_64/release/src/main.cu.o",
            "src/main.cu"
        ],
        "file": "src/main.cu"
    }
]
```

**Comment by waruqi** [comment]
try it again. `xmake update -s dev`

**Reply by owlx56** [reply]
![image](https://github.com/xmake-io/xmake/assets/105162283/a9e495fd-78a3-4d48-9710-cfce3555d583)

**Reply by waruqi** [reply]
try 

```
cd ..
xmake update -s dev
```

**Reply by owlx56** [reply]
![image](https://github.com/xmake-io/xmake/assets/105162283/ebcc120c-bed2-4384-b56c-3866faf4c799)

Xmake update fails for unknown reason.

**Reply by waruqi** [reply]
if your xmake is installed via apt/pacman, you need reinstall it via script.

```console
curl -fsSL https://xmake.io/shget.text | bash
source ~/.xmake/profile
xmake update -s dev
```

**Reply by owlx56** [reply]
After updating the issue has been solved:

![image](https://github.com/xmake-io/xmake/assets/105162283/d49f5720-8549-4ea6-bf2d-0ab6d172b9d0)
